mybatis.properties

```diff
@@ -2,13 +2,19 @@ category=External Analysers
 description=Rules to check SQL statements in MyBatis Mapper XML files.
 homepageUrl=https://github.com/donhui/sonar-mybatis
 archivedVersions=
-publicVersions=1.0.1
+publicVersions=1.0.1,1.0.2
```

Comment:
public versions can't overlap in compatibility. You need to move 1.0.1 to the archived list.

Reply:
I have moved version 1.0.1 to the archived list. Thanks.